.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish
+
+on:
+  schedule:
+    - cron: '0 0 1 */2 *'
+  workflow_dispatch:
```

Comment:
final question: shouldn't the `on` section include a restriction to the master branch, especially since the publishing is triggered automatically.